firmware/quadruna/VC/src/tasks.c

```diff
@@ -475,6 +515,44 @@ _Noreturn void tasks_runCanRx(void)
     }
 }
 
+_Noreturn void tasks_runLogging(void)
+{
+    static uint32_t count = 0;
+    for (;;)
+    {
+        if (!sd_functional())
+        {
+            osDelay(1000);
+            continue;
+        }
```

Comment:
dynamic sd card init? maybe move it from the init function to in here?

Reply:
This means that if the sd card fall off from the sd card slot, we should deschedule the task, and try again later.